.linting/duplicate-ids-whitelist.txt

```diff
@@ -135,8 +135,8 @@ _optAws 2
 _optCassandra 2
 _optDiscoUrl 2
 _optGundeck 2
-_optLogLevel 2
-_optLogNetStrings 2
+_optLogLevel 3
```

Comment:
We should probably talk about whether this is going to be a huge pain or not haha 😄

Reply:
I like the level of pain so far – it's just enough to make me want to spend time to rename All The Things (or fix the endpoint function mess, etc), but not enough to hate my life.

What we should really talk about is whether we are allowed to rename All The Things.